code/datums/components/crafting/recipes.dm

```diff
@@ -788,6 +788,16 @@
 				/obj/item/bikehorn = 1)
 	category = CAT_MISC
 
+/datum/crafting_recipe/flash_ducky
+	name = "Toy Rubber Duck Mine"
+	result = /obj/item/deployablemine/traitor/toy
+	time = 20
+	reqs = list(/obj/item/bikehorn/rubberducky = 1,
+				/obj/item/assembly/flash/handheld = 1,
+				/obj/item/stack/cable_coil = 2,
+				/obj/item/assembly/prox_sensor)
```

Comment:
```suggestion
				/obj/item/assembly/prox_sensor)
	blacklist = list(/obj/item/assembly/flash/handheld/strong)
```
It's rev flash. used no longer currently, but let's not it be consumed by it.

Reply:
Good call just in case.